nginx-controller/Dockerfile

```diff
@@ -5,7 +5,12 @@ FROM nginx:1.13.3
 RUN ln -sf /proc/1/fd/1 /var/log/nginx/access.log \
 	&& ln -sf /proc/1/fd/2 /var/log/nginx/error.log
 
-COPY nginx-ingress nginx/templates/nginx.ingress.tmpl nginx/templates/nginx.tmpl /
+COPY \
+    nginx-ingress \
```

Comment:
Could you change the first line to `COPY nginx-ingress  \`?

Reply:
OK. But I did it intentionally to indicate, that all lines have same semantic.